airbyte-integrations/connectors/source-prestashop/acceptance-test-config.yml

```diff
@@ -1,25 +1,36 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-prestashop:dev
-tests:
+acceptance_tests:
   spec:
-    - spec_path: "source_prestashop/spec.json"
-      # unfortunately timeout plugin takes into account setup code as well (docker setup)
-      timeout_seconds: 300
+    tests:
+      - spec_path: "integration_tests/spec.json"
+        # unfortunately timeout plugin takes into account setup code as well (docker setup)
```

Comment:
what do you mean by docker setup? build?

Reply:
in really from previous developer
good catch I just removed it
it seems we don't need it

Reply:
reverted back, we need it while docker containers with PrestaShop will up

Reply:
what's so special about this connector so it takes 5 minutes to go up? this is not a common issue, right?

Reply:
1. spec test starts with timer = 0
2. fixture connector_setup runs docker containers  
3. spec tests try to validate spec but 10 seconds already passed because of (2)